src/components/filterPanel/VideoSubtypeFilter.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { CheckboxFilter } from 'src/components/filterPanel/filter/CheckboxFilter';
+import { sortFilterOptions } from 'src/services/sortFilterOptions';
+import { FilterOption } from 'src/types/FilterOption';
+
+interface Props {
+  options: FilterOption[];
+  handleChange?: (filter: string, values: string[]) => void;
```

Comment:
Would this ever be not provided as a prop? If not then perhaps we should remove the optionality?

Reply:
Good catch ! It should always be here. I'll updated that